powerbi-docs/admin/troubleshoot-xmla-endpoint.md

```diff
@@ -130,9 +130,33 @@ When triggering a scheduled refresh or on-demand refresh in Power BI, Power BI t
 
 Overrides in [Refresh command (TMSL)](/analysis-services/tmsl/refresh-command-tmsl) allow users choosing a different partition query definition or data source definition for the refresh operation.
 
-## Errors in SSMS - Premium Gen 2
+## Errors on Premium Gen 2 capacity
```

Comment:
and PPU?

Reply:
PPU is mentioned under Gen2 in Gen 2 document: https://docs.microsoft.com/en-us/power-bi/admin/service-premium-per-user-faq. Also the original version mentions Gen2 instead of PPU + Gen2. So I prefer to keep Gen2, unless you have strong opinion.